frontend/src/utils/analysis-utils.ts

```diff
@@ -461,6 +462,9 @@ export async function loadFeaturesClientSide(
 }
 
 export function quoteAndEscapeCell(value: number | string) {
+  if (isUndefined(value)) {
```

Comment:
should quoteAndEscapeCell explicitely accept undefined values then? it might help simplify the typing logic in some places